frontend/src/viewer/components/path-element.ts

```diff
@@ -333,13 +359,116 @@ export class PathCtrlElement extends connect(store)(LitElement) {
     }
   }
 
+  protected launchScoring(): void {
+    if (this.tracks && this.currentTrack !== null && this.tracks[this.currentTrack]) {
+      const fixes: BRecord[] = [];
+      for (let i = 0; i < this.tracks[this.currentTrack].fixes.lat.length; i++)
+        // Keep this to the bare minimum needed for igc-xc-score
+        fixes[i] = {
+          timestamp: this.tracks[this.currentTrack].fixes.ts[i],
+          latitude: this.tracks[this.currentTrack].fixes.lat[i],
+          longitude: this.tracks[this.currentTrack].fixes.lon[i],
+          pressureAltitude: this.tracks[this.currentTrack].fixes.alt[i],
+          gpsAltitude: this.tracks[this.currentTrack].fixes.alt[i],
+          valid: true,
```

Comment:
Do we need to specify all the record that have a constant value ? (valid -> enl)